src/run/docker-compose.ts

```diff
@@ -53,6 +53,10 @@ export class DockerCompose {
         );
 
         this.logFile = join(logsDir, `compose.out.${dateLabel}.txt`);
+
+        if (!existsSync(this.logFile)) {
+            closeSync(openSync(this.logFile, "w"));
```

Comment:
This is a little unclear how it fixes the issue you have highlighted - it probably needs at the very least needs a comment. How has this been tested? This said, I am not sure what the issue is since my chs-dev is outputting `0` prefixed dates
![Screenshot 2024-11-06 at 13 19 54](https://github.com/user-attachments/assets/ff9519d3-450f-40fd-b396-4be48058e200)

Might I suggest removing some of the older logs which are in the incorrect format and retry? Older versions of `chs-dev` did indeed have incorrect logging behaviours which have subsequently been fixed so worth testing with the later versions in case it is as a result of that (`chs-dev sync -v latest` will take care of that)

Reply:
Hi Joel
I've tested this locally quickly and hackily, I'll attach some evidence here now but happy to flesh it out as required. To be honest I too am unclear exactly what's going on here however this issue is resolved for me with this approach.

In the below image I'm using my existing chs-dev installation. If I run the sync command it appears to be happy with the version
![image](https://github.com/user-attachments/assets/79b3528c-3213-4cc3-bc66-30468e7c5ff1)

And here I'm using my local dev build with these changes
![image](https://github.com/user-attachments/assets/dbd9f6e5-a3a8-465f-accc-c05ceb053f31)

Reply:
Your main version `1.3.17` looks quite out of date and like it has the changes in `https://github.com/companieshouse/chs-dev/releases/tag/1.3.22` which should resolve the issues you are experiencing. The fact you have the changes from main have probably masked the issue. Try running `chs-dev sync -v latest` and accept the reinstall and you'll find the issue will be resolved.

Reply:
Can confirm that's resolved it and the change is pointless, was just a matter of using the latest code 😄  Will close this and distribute the update process to people I'm aware of this impacting. Thanks!